effectsize_example.Rmd

```diff
@@ -97,9 +97,9 @@ library(forcats)    # for fct_...()
 library(broom)      # for tidy()
 library(ggstance)   # for geom_pointrangeh()
 ```
+This function is for formatting numbers:
```

Comment:
The `boilerplate` code block following this is not rendered in the output (due to `include = FALSE`), so this comment will not point at anything. I think for `format_num` this is fine, since `format_num` is not really used in any visible code, just in the inline R code chunks to make text output. That was my rationale for putting it in a non-rendered boilerplate block.

I do like the move of set.seed() into the data generation block and out of the boilerplate block though.